samples/legacy_support/All_ID_Mapping_Original.csv

```diff
@@ -915,7 +915,7 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 913,Military (Space) : Space Launched Vehicle (SLV),S-P-L-----,05111900,,,,,Point,,,
 914,Military (Space) : Space Vehicle : Crewed,S-P-V-----,S-P-V-----,,,,,Point,,,
 915,Command and Control : Civil Affairs,S-F-GC----,10110200,,,,,Point,,,
-916,Command and Control : Military Information Support (MISO),S-F-GP----,10110600,,,,,Point,,,
+916,Command and Control : Military Information Support (MISO),S-F-GP----,40130200,,,,,Point,,,
```

Comment:
@csmoore : mapping from one symbol set into another is not my recommended approach for this.  I'd suggest creating a new S-F-GP----.svg file that looks the same as 40130200.